### PR TITLE
BIO-A or OCC-AUTH before in person PIN reset

### LINKED ISSUE #399: What happens if you reset the PIN and the biometric match fails?
### All Fields Are Required
**Organization Name (N/A, if individual)**: DHS

**Organization Type (see below for codes)**: 1 - Federal
 
**Reference (Include section/paragraph or pdf line number)**: Line 999-1000
 
**Comment (Include rationale for comment)**: "...before providing the reset PIV Card back to the cardholder..."
 
**Suggested Change**: "What happens if you reset the PIN and the biometric match fails?

Recommend reducing risk.  Recommend new language ""…before resetting the PIV Card's PIN…""  You really should know who is sitting with you prior to enabling the card for operational use again.

Recommended change is consistent with 1016-1018."
 
---
 
Organization Type: 1 = Federal, 2 = Industry, 3 = Academia, 4 = Self, 5 = Other

### PR DESCRIPTION
**Pull request by gfiumara**
Closes #399.

### CODE REVIEW

**Review by jricher — APPROVED** [review]

File: _FIPS201/requirements.md [hunk 1]
```diff
@@ -501,8 +501,8 @@ PIN reset **MAY** be performed in person at an issuing facility, at a kiosk oper
 remotely via a general computing platform or a supervised remote identity proofing station:
 
 In person
-: When PIN reset is performed in person at the issuing facility, before providing the reset PIV Card
-    back to the cardholder, the issuer **SHALL** perform a biometric verification to ensure that the cardholder's
+: When PIN reset is performed in person at the issuing facility, before
+    resetting the PIV Card's PIN, the issuer **SHALL** first perform a biometric verification to ensure that the cardholder's
     biometric characteristics elicit a positive biometric verification decision when compared to biometric data records stored in the PIV enrollment record or when compared to the biometric data records on the PIV Card using the BIO-A or OCC-AUTH authentication mechanisms. In cases where a negative biometric verification decision is returned or the cardholder's biometric characteristics are not successfully acquired, the cardholder **SHALL** provide the PIV Card
```
Comment (hferraio) [inline _FIPS201/requirements.md:506]:
```suggestion
: When PIN reset is performed in person at the issuing facility, before
    resetting the PIV Card's PIN, the issuer **SHALL** first perform a biometric verification to ensure that the cardholder's
    biometric characteristics elicit a positive biometric verification decision when compared to biometric data records stored in the PIV enrollment record or when compared to the biometric data records on the PIV Card using OCC-AUTH authentication mechanisms. Alternatively, BIO-A authentication mechanism **MAY** be used as biometric verification after the card has been reset and while the card is in the operator's custody. The operator then hands the reset PIV Card to the cardholder. In cases where a negative biometric verification decision is returned or the cardholder's biometric characteristics are not successfully acquired, the cardholder **SHALL** provide the PIV Card
```
```suggestion
: When PIN reset is performed in person at the issuing facility, before
    resetting the PIV Card's PIN, the issuer **SHALL** first perform a biometric verification to ensure that the cardholder's
    biometric characteristics elicit a positive biometric verification decision when compared to biometric data records stored in the PIV enrollment record or when compared to the biometric data records on the PIV Card using the BIO-A or OCC-AUTH authentication mechanisms. In cases where a negative biometric verification decision is returned or the cardholder's biometric characteristics are not successfully acquired, the cardholder **SHALL** provide the PIV Card
```
```suggestion
: When PIN reset is performed in person at the issuing facility, before
    resetting the PIV Card's PIN, the issuer **SHALL** first perform a biometric verification to ensure that the cardholder's
    biometric characteristics elicit a positive biometric verification decision when compared to biometric data records stored in the PIV enrollment record or when compared to the biometric data records on the PIV Card using the BIO-A or OCC-AUTH authentication mechanisms. In cases where a negative biometric verification decision is returned or the cardholder's biometric characteristics are not successfully acquired, the cardholder **SHALL** provide the PIV Card
```
```suggestion
: When PIN reset is performed in person at the issuing facility, before
    resetting the PIV Card's PIN, the issuer **SHALL** first perform a biometric verification to ensure that the cardholder's
    biometric characteristics elicit a positive biometric verification decision when compared to biometric data records stored in the PIV enrollment record or when compared to the biometric data records on the PIV Card using the BIO-A or OCC-AUTH authentication mechanisms. In cases where a negative biometric verification decision is returned or the cardholder's biometric characteristics are not successfully acquired, the cardholder **SHALL** provide the PIV Card
```

**Review by hferraio — CHANGES_REQUESTED** [review]
BIO-A authentication mechanism is a two factor authentication method that first takes the PIN (verify), then releases the biometric on the card to be match off-card.  This method does not work on a card that is not reset yet.  Hence the changes.  WDYT?

**Comment by gfiumara** [comment]
@hferraio Good point regarding BIO-A. IMO, this doesn't address what the requester asked for. We should potentially *not* allow BIO-A in this case? I will make the change to remove it _before_ now.

**Comment by hferraio** [comment]
@gfiumara: Well, BIO-A and enrollment is the established method (as per FIPS 201-2). If we get rid of it, I am afraid we pull the rug underneath some implementation (hence my email to DoD and the MSO).  I thought our resolution was to allow both (OCC and BIO-A)?

**Comment by jimfenton** [comment]
I'm guessing that the common case for PIN reset is forgotten PIN. BIO-A would not be usable in that case, so I don't think it would be pulling the rug out too seriously. But I suppose that someone might change their PIN because someone saw them entering it, and BIO-A would still be usable in that case, so I don't see a problem with allowing it.

**Review by jimfenton — COMMENTED** [review]
Generally like the changes, but see previous comment.

**Comment by hferraio** [comment]
@jimfenton:  What has been possible until now ( per FIPS 201-2), is to reset the PIN **first** - now the BIO-A AuthN is available again since the PIN is unlocked.  Do BIO-A, then security guard/staff hands the card to the cardholder only after biometric comparison matched. (This is a in-person scenario we are discussing).

**Comment by jglosx3** [comment]
Closed

**Comment by jglosx3** [comment]
Closed because issue is declined

**Comment by gfiumara** [comment]
Re-opening per request:
> The DoD performs the biometric comparison before the resetting of the PIN at this time.

File: _FIPS201/requirements.md [hunk 2]
```diff
@@ -501,9 +501,9 @@ PIN reset **MAY** be performed in person at an issuing facility, at a kiosk oper
 remotely via a general computing platform or a supervised remote identity proofing station:
 
 In person
-: When PIN reset is performed in person at the issuing facility, before providing the reset PIV Card
-    back to the cardholder, the issuer **SHALL** perform a biometric verification to ensure that the cardholder's
-    biometric characteristics elicit a positive biometric verification decision when compared to biometric data records stored in the PIV enrollment record or when compared to the biometric data records on the PIV Card using the BIO-A or OCC-AUTH authentication mechanisms. In cases where a negative biometric verification decision is returned or the cardholder's biometric characteristics are not successfully acquired, the cardholder **SHALL** provide the PIV Card
+: When PIN reset is performed in person at the issuing facility, before
+    resetting the PIV Card's PIN, the issuer **SHALL** first perform a biometric verification to ensure that the cardholder's
```
Comment (gfiumara) [inline _FIPS201/requirements.md:505]:
```suggestion
    resetting the PIV Card's PIN, the issuer **SHALL** perform a biometric verification to ensure that the cardholder's
```

File: _FIPS201/requirements.md [hunk 3]
```diff
@@ -501,9 +501,9 @@ PIN reset **MAY** be performed in person at an issuing facility, at a kiosk oper
...
+    biometric characteristics elicit a positive biometric verification decision when compared to biometric data records stored in the PIV enrollment record or when compared to the biometric data records on the PIV Card using the OCC-AUTH authentication mechanism. In cases where a negative biometric verification decision is returned or the cardholder's biometric characteristics are not successfully acquired, the cardholder **SHALL** provide the PIV Card
```
Comment (gfiumara) [inline _FIPS201/requirements.md:506]:
```suggestion
    biometric characteristics elicit a positive biometric verification decision when compared to biometric data records stored in the PIV enrollment record or when compared to the biometric data records on the PIV Card using the BIO-A or OCC-AUTH authentication mechanisms. In cases where a negative biometric verification decision is returned or the cardholder's biometric characteristics are not successfully acquired, the cardholder **SHALL** provide the PIV Card
```

**Pull request closed without merging**